spec/requests/create_heritage_spec.rb

```diff
@@ -66,18 +66,27 @@
       expect(heritage["services"][0]["command"]).to eq "nginx"
       expect(heritage["services"][0]["force_ssl"]).to eq true
       expect(heritage["services"][0]["reverse_proxy_image"]).to eq "org/custom_revpro:v1.2"
-      expect(heritage["services"][0]["port_mappings"][0]["lb_port"]).to eq 3333
```

Comment:
I need to update this because it depends on orders of `port_mappings`. But contents for test is the same as before.

Reply:
make sense. i had to deal with that too